hw/ip/sram_ctrl/dv/sram_ctrl_base_sim_cfg.hjson

```diff
@@ -102,6 +102,8 @@
     {
       name: "{variant}_access_during_key_req"
       uvm_test_seq: sram_ctrl_access_during_key_req_vseq
+      // Need to make sure no sram access when we just request new key or init, so use zero_delays
```

Comment:
Perhaps this is better explained in the sequences?

Reply:
Let me explain it in sequence as well.